<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCollegueView</class>
 <widget class="QWidget" name="AddCollegueView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1294</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1294</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnAddCollegue">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>323</y>
     <width>340</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnAddCollegue
{	
 	border:1px solid #ffffff;
	color:white;
	background-color:transparent;
 }

QPushButton#btnAddCollegue:hover
{
 	color:#0992b1;
 } 

QPushButton#btnAddCollegue:pressed
{
 	color:#0992b1;
	border: 1px solid #0992b1; 
} </string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="ColleaguesLineEditd" name="txtFirstName">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>105</y>
     <width>340</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:ffffff;
border:none;
border-bottom:1px solid #909090;
background-color:transparent;
padding-left:7px;
padding-bottom:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ColleaguesLineEditd" name="txtLastName">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>162</y>
     <width>340</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:ffffff;
border:none;
border-bottom:1px solid #909090;
background-color:transparent;
padding-left:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ColleaguesLineEditd" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>219</y>
     <width>340</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:ffffff;
border:none;
border-bottom:1px solid #909090;
background-color:transparent;
padding-left:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ColleaguesLineEditd" name="txtPhone">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>276</y>
     <width>340</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:ffffff;
border:none;
border-bottom:1px solid #909090;
background-color:transparent;
padding-left:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ColleaguesLabel" name="lblFirstName">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>113</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#909090;</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="ColleaguesLabel" name="lblLastName">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#909090;</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="ColleaguesLabel" name="lblEmail">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>227</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#909090;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="ColleaguesLabel" name="lblPhone">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>284</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#909090;</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIncorrectInput">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>380</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
 opacity: 0;</string>
   </property>
   <property name="text">
    <string>Input correct all filesds</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCollegues">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Collegues</string>
   </property>
  </widget>
  <widget class="ColleaguesLabel" name="lblBackGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1294</width>
     <height>600</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblBackGround</zorder>
  <zorder>btnAddCollegue</zorder>
  <zorder>txtFirstName</zorder>
  <zorder>txtLastName</zorder>
  <zorder>txtEmail</zorder>
  <zorder>txtPhone</zorder>
  <zorder>lblFirstName</zorder>
  <zorder>lblLastName</zorder>
  <zorder>lblEmail</zorder>
  <zorder>lblPhone</zorder>
  <zorder>lblIncorrectInput</zorder>
  <zorder>lblCollegues</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColleaguesLineEditd</class>
   <extends>QLineEdit</extends>
   <header location="global">CommonUiControllers\Views\Colleagues\ColleaguesLineEditd.h</header>
  </customwidget>
  <customwidget>
   <class>ColleaguesLabel</class>
   <extends>QLabel</extends>
   <header location="global">CommonUiControllers\Views\Colleagues\ColleaguesLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
